scripts/quickstart.sh

```diff
@@ -93,16 +93,35 @@ conf() {
     fi
 
     if grep -q "$1" $env_file; then
-        sed -i "s/$1=.*/$1=$2/" $env_file
+        sed -i "s|$1=.*|$1=$2|" $env_file
     else
         echo "$1=$2" >> $env_file
     fi
+
+    # Extract SUPERBLOCKS_AGENT_DOMAIN from SUPERBLOCKS_AGENT_HOST_URL
+    if [ "$1" = "SUPERBLOCKS_AGENT_HOST_URL" ]; then
+        if [[ $2 == http* ]]; then
+            domain=$(awk -F/ '{print $3}' <<<"$2")
+            conf "SUPERBLOCKS_AGENT_DOMAIN" $domain
```

Comment:
We also have "__SUPERBLOCKS_AGENT_DOMAIN" that must always point to the Superblocks Cloud domain. Perhaps this variable is too similar to that one?

Reply:
I didn't know that one. Do you have any suggestion here?

Reply:
Renamed "SUPERBLOCKS_AGENT_DOMAIN" => "SUPERBLOCKS_AGENT_CUSTOM_DOMAIN"